File: server/dotnet/sdk-dotnet-example/Infrastructure/Mappers/PaymentDetails/HostedCheckoutSpecificOutputMapper.cs
using HostedCheckoutSpecificOutputSdk = OnlinePayments.Sdk.Domain.HostedCheckoutSpecificOutput;
using HostedCheckoutSpecificOutputDto = Business.Domain.Payments.PaymentDetails.HostedCheckoutSpecificOutput;

namespace Infrastructure.Mappers.PaymentDetails;

public static class HostedCheckoutSpecificOutputMapper
{
    public static HostedCheckoutSpecificOutputDto Map(HostedCheckoutSpecificOutputSdk? response)
    {
        return new()
        {
            Variant = response?.Variant,
            HostedCheckoutId = response?.HostedCheckoutId
        };
    }
}

File: server/php/src/Infrastructure/Mappers/PaymentDetails/HostedCheckoutSpecificOutputMapper.php
<?php

namespace OnlinePayments\ExampleApp\Infrastructure\Mappers\PaymentDetails;

use OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails\HostedCheckoutSpecificOutput as HostedCheckoutSpecificOutputDomain;
use OnlinePayments\Sdk\Domain\HostedCheckoutSpecificOutput as HostedCheckoutSpecificOutputSdk;

final class HostedCheckoutSpecificOutputMapper
{
    public static function mapFromSdkResponse(?HostedCheckoutSpecificOutputSdk $response): ?HostedCheckoutSpecificOutputDomain
    {
        if ($response === null) {
            return null;
        }

        $output = new HostedCheckoutSpecificOutputDomain();
        $output->hostedCheckoutId = $response->getHostedCheckoutId();
        $output->variant = $response->getVariant();

        return $output;
    }
}
